kwok/kwok_container_template.yml

```diff
@@ -0,0 +1,15 @@
+# Save the output of this file and use kubectl create -f to import
+# it into Kubernetes.
+#
+# Created with podman-4.5.1
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ .PodName}}
+spec:
+  containers:
+  - name: kwok
+    image: localhost/kwok:latest
+    ports:
+    - containerPort: 8080
+      hostPort: 8080
```

Comment:
yes, Else I would need to make changes to buildAndRunPod()

Reply:
We can set a specific pod-name and ignore the templating operation (as we do with the ports)
but besides making the it clear that we don't need to run it concurrently, there are no other benefits, so it can stay as-is...